### ISSUE TITLE
Does the updater also update the web ui?

### ISSUE THREAD

**Issue by Kuurusch** [ipfs/ipfs-update#52]
The title says it already. Does the updater also update the web ui? Because in the last relaese rc4, the web ui does still not working

**Comment by whyrusleeping** [comment]
@Kuurusch It should be working in rc4. What problems are you having?

cc @dignifiedquire

**Comment by Kuurusch** [comment]
Hm, so I work on OSX and in the web ui the swarm is still zero. No other peers.

**Comment by whyrusleeping** [comment]
@Kuurusch thats quite odd. @dignifiedquire any ideas here? You were testing this on OSX before the release.

@Kuurusch Does the developer console show any errors?

**Comment by Kuurusch** [comment]
@whyrusleeping Hm you mean the output of the daemon?

**Comment by Kuurusch** [comment]
@whyrusleeping Have you a idea how I can debug this situation? I dont understand exactly where the webui is stored. Or is it integrated in the API-bin? Could it be, that it references to an old webui path and the new one has an other path?

**Comment by Kubuxu** [comment]
Have you tried accessing `localhost:5001/webui`, the hash of webui will be different than before and this should redirect you to new hash.

**Comment by Kuurusch** [comment]
So wehen I accessing localhost:5001/webui it redirects me to http://localhost:5001/ipfs/QmPhnvn747LqwPYMJmQVorMaGbMSgA7mRRoyyZYz3DoZRQ/#/

**Comment by Kuurusch** [comment]
Ou, I've recognised, that now it works :) Hear where I work I'm in the clear-net. At home I'm in Hyperboria. Maybe that was the problem :X But before that worked great :/

**Comment by Kuurusch** [comment]
@Kubuxu Ou, edit! it's strange, so in Hyperboria the Webgui works not but in the terminal I have connections?

**Comment by Kubuxu** [comment]
Can you rephrase that last comment. It is a bit unclear.

**Comment by whyrusleeping** [comment]
@Kuurusch by the developer console i mean the one in your browser. You can press ctrl+shift+J to open it up on chrome.

**Comment by Kuurusch** [comment]
@Kubuxu Ou sorry, I was a little confusing. Now I understand it more. So it was really only the hash. I had the link to the webui stored with the old hash in my bookmarks and because of that I had this problems. When I use only localhost:5001/webui it works and redirekts to the right hash. Maybe it would be good to find a way, that it don't show the hash in the address?

**Comment by Kubuxu** [comment]
Please report it as separate issue to webui (first check if it isn't already reported).
